xee/ext.py

```diff
@@ -463,7 +463,11 @@ def image_to_array(
     Returns:
       An numpy array containing the pixels computed based on the given image.
     """
-    image = image.unmask(self.mask_value)
+    image = (
+        ee.Image(self.mask_value)
+        .rename(image.bandNames().getInfo()[0])
```

Comment:
What does `.rename` do here?

`ee.Image(self.mask_value)` should produce an image with 1 band. From the EE documentation, "If either of images has only 1 band, it is replicated to match the number of bands in the other image."

We shouldn't have to rename the bands for the blend to work properly. I'd like to avoid doing a `.getInfo()` call here because it's expensive.

Reply:
The behavior of the blend is that if both images have the same bands, it returns the name of the first image's band in the output.

In our case:

 - ee.Image(ee_mask_value) has only 1 band.
   - When the user's image has a single band, it returns the name of the first image's band in the output(name of the ee.Image(ee_mask_value)). Therefore, renaming is necessary in this case.
   - In the context of multiband images, renaming is not necessary.

Thus, renaming is **required** when the user image has only 1 band.

Code Without renaming:
```
image1 = ee.Image("projects/anthromet-prod/assets/1km_uk_nimrod_composite_rainfall_radar/201701010140_nimrod_ng_radar_rainrate_composite_1km_UK")
updated_image1 = ee.Image(ee_mask_value).blend(image1)
print(updated_image1.getInfo()['bands'])

image2 = ee.Image('COPERNICUS/S1_GRD/S1A_IW_GRDH_1SDV_20200113T094313_20200113T094338_030780_0387B2_0E67')
updated_image2 = ee.Image(ee_mask_value).blend(image2)
print(updated_image2.getInfo()['bands'])
```